src/care/forall.h

```diff
@@ -193,6 +228,10 @@ namespace care {
 #else
```

Comment:
This is totally unrelated to this PR, but I noticed we have a few lines above

forall(RAJA::hip_exec<CARE_CUDA_BLOCK_SIZE, CARE_CUDA_ASYNC>{},
             fileName, lineNumber, start, end, body);

above. In the future we should remember to either have CARE_GPU_BLOCK_SIZE (just rename it) or have separate CARE_CUDA_BLOCK_SIZE and CARE_HIP_BLOCK_SIZE

Reply:
Yeah, we'll need to clean that up. Thanks for pointing that out.

Reply:
I created an issue so that we remember to fix this.